etc/smashbox.conf.template

```diff
@@ -29,7 +29,7 @@ oc_number_test_groups=1
 
 # password for test accounts: all test account will have the same password
 # if not set then it's an error
-oc_account_password=""
+oc_account_password="demo"
```

Comment:
Since the value must be set, it's either provide a default value here, or make it to be required. I'd rather provide a default to make the least noise as possible.